### DISCUSSION TITLE
moving a model from the latest version of PyG to an old version

### DISCUSSION THREAD

**Discussion by davidireland-iso** [pyg-team/pytorch_geometric#3609] [Q&A] [answered]
Similar to my question [here](https://github.com/pyg-team/pytorch_geometric/discussions/3517), is it possible to take a trained model from the latest version and use it in the old version? If it helps, the PyG layers I use are `SAGEConv`, `TopKPooling` and `global_mean/max_pool`. When I tried to load the model in directly I got the error 'no module named `torch_geometric.nn.dense.linear`' so I am guessing it might not be possible. 

Perhaps it is easier for me to install the older version of PyG on the machine I am training on? however I was unable to find the pip/conda command to install a previous version.

**Comment by rusty1s** [comment, marked as answer]
Does it work when only saving and loading the `state_dict`?

**Reply by davidireland-iso** [reply]
I believe that has worked, thanks a lot!
